### ISSUE TITLE
Icon Buttons aren't generating alt text

### ISSUE THREAD

**Issue by jasononeil** [cultureamp/kaizen-design-system#1289]
In our Button component, the `renderIcon()` function looks like this:

```
const renderIcon = (icon: React.SVGAttributes<SVGSymbolElement>) => (
  <span className={styles.iconWrapper}>
    <Icon icon={icon} role="presentation" />
  </span>
)
```

This is used both for icons on buttons with labels, and on the `IconButton`, which doesn't have a label. The result is that the Icon component has a `aria-labelledby` attribute with a UUID, but no element with that ID exists, so no alt text is available for the icon.

**Comment by dougmacknz** [comment]
The `IconButton` is actually okay by way of an `aria-label` that exists on the `<button>`, the `Icon` usage in `IconButton` has role="presentation", so it shouldn't have the `aria-labelledby` property.

Here's the bug in Icon.tsx~L61: `aria-labelledby` is still being added even when `!isMeaningfulImg`

![image](https://user-images.githubusercontent.com/1811583/120424527-e5618a00-c3af-11eb-95d9-1d55324e4b92.png)
